code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -33,4 +33,5 @@
 
 	var/coughedtime = null // should only be useful for carbons as the only thing using it has a carbon arg.
 
-	var/willfully_sleeping = 0
\ No newline at end of file
+	var/willfully_sleeping = 0
+	var/consume_nutrition_from_air = FALSE // used by Diona
```

Comment:
Isn't there a "diona datum" thing where this variable could be off-loaded to?

Reply:
Yes, there is. "diona/datum".

Reply:
It is also used by Diona nymph which is of carbon type and doesn't have diona datum iirc

Reply:
_Aaaaaaaaaaaaaaaaaaaaaa._

Reply:
I repeat my statement. _Aaaaaaaaaaaaaaaaaaaaaaaa._